mlflow/tracking/client.py

```diff
@@ -131,8 +131,8 @@ def log_metric(self, run_id, key, value, timestamp=None):
         Log a metric against the run ID. If timestamp is not provided, uses
         the current timestamp.
         """
-        _validate_metric_name(key)
         timestamp = timestamp if timestamp is not None else int(time.time())
+        _validate_metric(key, value, timestamp)
```

Comment:
Should we call this in `log_batch` as well?

Reply:
Oops yeah lol...